### PR TITLE
Allow customLabels for the exporter deployment

### PR DESCRIPTION
**Pull request by dmyerscough**
#### What this PR does / why we need it:

This PR is to allow custom labels to be set on the `exporter` deployment. The reason you might want to add custom labels on the exporter vs exporter/operator is to instruct a scraper to scrape the metrics endpoint. The current custom labels set the label on both the operator and exporter deployment.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by dmyerscough** [comment]
Does this PR look reasonable to you?

cc @Jasstkn

File: charts/litmus-core/templates/_helpers.tpl [hunk 1]
```diff
@@ -41,8 +41,10 @@ app.kubernetes.io/part-of: {{ template "litmus.name" . }}
 app.kubernetes.io/version: "{{ .Chart.Version }}"
 helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }} 
 litmuschaos.io/version: {{ .Chart.AppVersion }}
-{{- if .Values.customLabels }}
+{{- if and (.Values.customLabels) (eq .deployment "operator") }}
 {{ toYaml .Values.customLabels }}
```
Comment (gdsoumya) [inline charts/litmus-core/templates/_helpers.tpl:45]: This one is a custom set of labels that's applied to all resources and not just the operator deployment. So maybe remove the deplyoment lable check.

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: charts/litmus-core/templates/_helpers.tpl (hunk 1 above)
Comment (dmyerscough) [reply charts/litmus-core/templates/_helpers.tpl:45]: @gdsoumya just updated this.

**Comment by dmyerscough** [comment]
Ill open a new request to address this

**Pull request closed without merging**